lua/autorun/server/bounties.lua

```diff
@@ -0,0 +1,78 @@
+
+hook.Add("PlayerDeath","PlayerDeath",function(ply,infl,atk)
+	local bounty = tonumber(ply:GetPData("bounty",0))
+	if bounty > 0 and ply != atk and type(atk) == "Player" then
+		atk:PS_GivePoints(bounty)
```

Comment:
Set the bounty to zero before giving points. If there was some bug where BotSayGlobal caused an error this would be an infinite points exploit